### PR TITLE
Testing Django relationships

### PR DESCRIPTION
**Pull request by abhaykatheria**
this commit adds tests for -
 > one to one field
 > many to many field
 > foreign key field (many to one)

Signed-off-by: abhaykatheria <abhay.katheria1998@gmail.com>

### CODE REVIEW

**Review by GuptaManan100 — APPROVED** [review]
Amazing work ✌️ !

**Review by mcronce — APPROVED** [review]
This looks real good to me.  One suggestion, but it's fine to merge as-is.

File: frameworks/python/django/src/testApp/models.py [hunk 1]
```diff
@@ -50,3 +49,47 @@ class Server(models.Model):
...
     def __str__(self):
         return self.name
+
+class HQAddress(models.Model):
+    house_no = models.CharField(max_length=100)
+    street = models.CharField(max_length=100)
+    city = models.CharField(max_length=100)
+
+    def __str__(self):
+        return "%s ,%s ,%s" % (self.house_no, self.street, self.city)
```
[the remark this answers is not in the document]
Comment (abhaykatheria) [reply frameworks/python/django/src/testApp/models.py]: done 👍🏾

**Comment by abhaykatheria** [comment]
@mcronce  Can you take a look now, please 😅  ?

**Review by mcronce — APPROVED** [review]
Looks great :) thanks!

**Pull request merged**